definitions/procedures/backup/offline/foreman_db.rb

```diff
@@ -0,0 +1,48 @@
+module Procedures::Backup
+  module Offline
+    class ForemanDB < ForemanMaintain::Procedure
+      metadata do
+        description 'Backup Foreman DB offline'
```

Comment:
Not sure why it is failing 
```
Backup Foreman DB offline:
Backup of /var/lib/pgsql/data/ is not supported for remote databases. Doing postgres dump instead...
/ Getting Foreman DB dump                                             [FAIL]
Failed executing PGPASSWORD='5yPDJmCuHhkxDiL9GHA8GrbBNMxXid6J' pg_dump -h localhost  -p 5432 -U foreman -Fc foreman > /tmp/foreman-backup-2018-04-17-02-01-53/foreman.dump, exit status 1
--------------------------------------------------------------------------------
```
```
E, [2018-04-17 02:32:07-0400 #62030] ERROR -- : Failed executing PGPASSWORD='5yPDJmCuHhkxDiL9GHA8GrbBNMxXid6J' pg_dump -h localhost  -p 5432 -U for
eman -Fc foreman > /tmp/foreman-backup-2018-04-17-02-01-53/foreman.dump, exit status 1 (ForemanMaintain::Error::ExecutionError)
/root/foreman_maintain/lib/foreman_maintain/utils/command_runner.rb:50:in `execution_error'
/root/foreman_maintain/lib/foreman_maintain/concerns/system_helpers.rb:67:in `execute!'
/root/foreman_maintain/lib/foreman_maintain/concerns/base_database.rb:42:in `dump_db'
/root/foreman_maintain/definitions/procedures/backup/offline/foreman_db.rb:30:in `block in run'
/root/foreman_maintain/lib/foreman_maintain/reporter/cli_reporter.rb:135:in `with_spinner'
/root/foreman_maintain/definitions/procedures/backup/offline/foreman_db.rb:29:in `run'
/root/foreman_maintain/lib/foreman_maintain/executable.rb:139:in `__run__'
/root/foreman_maintain/lib/foreman_maintain/runner/execution.rb:83:in `block (2 levels) in run'
/root/foreman_maintain/lib/foreman_maintain/runner/execution.rb:107:in `capture_errors'
/root/foreman_maintain/lib/foreman_maintain/runner/execution.rb:82:in `block in run'
/root/foreman_maintain/lib/foreman_maintain/runner/execution.rb:101:in `with_metadata_calculation'
/root/foreman_maintain/lib/foreman_maintain/runner/execution.rb:81:in `run'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:114:in `run_step'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:103:in `run_steps'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:96:in `execute_scenario_steps'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:49:in `run_scenario'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:31:in `block in run'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:30:in `each'
/root/foreman_maintain/lib/foreman_maintain/runner.rb:30:in `run'
/root/foreman_maintain/lib/foreman_maintain/cli/base.rb:47:in `run_scenario'
/root/foreman_maintain/lib/foreman_maintain/cli/backup_command.rb:38:in `perform_backup'
/root/foreman_maintain/lib/foreman_maintain/cli/backup_command.rb:97:in `execute'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
/root/foreman_maintain/lib/foreman_maintain/cli.rb:25:in `run'
/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
```

Reply:
It could not find that dump due to cleanup..will share an instance with you

Reply:
Was this comment resolved?

Reply:
Yes, it was